keps/sig-scheduling/1258-default-pod-topology-spread/README.md

```diff
@@ -259,13 +272,200 @@ To ensure this feature to be rolled out in high quality. Following tests are man
 - [x] Score extension point implementation. Add support for `maxSkew`.
 - [x] Filter extension point implementation.
 - [x] Disabling `SelectorSpread` when the feature is enabled.
-- [x] Unit, Integration and benchmark test cases mentioned in the [Test Plan](#test-plan).
+- [x] Unit and benchmark test cases mentioned in the [Test Plan](#test-plan).
+
+#### Beta (v1.20):
+
+- [ ] Finalize implementation:
+  - [ ] Map `SelectorSpreadingPriority` to `PodTopologySpread` when using Policy API.
+  - [ ] Provide knob for disabling the k8s default constraints.
+- [ ] Integration tests.
+- [ ] Verify conformance tests passing.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: `DefaultPodTopologySpread`
+    - Components depending on the feature gate: `kube-scheduler`
+  - [x] Other
+    - Describe the mechanism:
+
+      Explicitly disable default spreading constraints for the `PodTopologySpread` plugin in the kube-scheduler config (passed via `--config` command line flag):
+
+      ```yaml
+      apiVersion: kubescheduler.config.k8s.io/v1beta1
+      kind: KubeSchedulerConfiguration
+      profiles:
+        - pluginConfig:
+          - name: PodTopologySpread
+            args:
+              disableDefaultConstraints: true
+      ```
+
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+
+      Only kube-scheduler needs to be restarted.
+
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+      No
+
+* **Does enabling the feature change any default behavior?**
+
+  Yes. Users might experience more spreading of Pods among Nodes and Zones in certain topology distributions.
```

Comment:
What are the default spreading params? How do we group pods for spreading by default?

Reply:
Also - is the new default spreading preferred or forced (i.e. predicate or priority)?

Reply:
It's documented in the KEP already #default-constraints and it continues to be scoring (priority)

Reply:
Got it - can you please cross-link here?